Source/Features/Store/Interactor/GetStorePagesInteractor.swift

```diff
@@ -15,49 +15,38 @@ import Harmony
 
 struct GetStorePagesInteractor {
     private let executor: Executor
-    private let getStoreNotificationInteractor: Interactor.GetByQuery<[String: StorePage]>
+    private let getStoreNotificationInteractor: Interactor.GetByQuery<StorePage>
     
     init(
         executor: Executor,
-        getStoreNotificationInteractor: Interactor.GetByQuery<[String: StorePage]>
+        getStoreNotificationInteractor: Interactor.GetByQuery<StorePage>
     ) {
         self.executor = executor
         self.getStoreNotificationInteractor = getStoreNotificationInteractor
     }
     
     func execute(storePredicate: StorePredicate,
-                 cursorPredicate: CursorPredicate,
+                 pagePredicate: StorePagePredicate,
                  userQuery: UserQuery,
                  in executor: Executor? = nil) -> Future<StorePage> {
         execute(
-            contexts: [
-                StoreContext("data", storePredicate, cursorPredicate)
-            ],
```

Comment:
One thing I didn't understand is why we supported multiple contexts here. There always ever was one context used. I've went ahead and simplified that part of the logic.